README.md

```diff
@@ -91,13 +148,13 @@ capabilities are required to the `extraCapabilities` object, and then call the p
 shown below
 
 ```java
-	@BeforeSuite(alwaysRun = true)
-	public void beforeSuite() throws InvalidBrowserException {
-		extraCapabilities = new DesiredCapabilities();
-		extraCapabilities.setCapability(InternetExplorerDriver.INTRODUCE_FLAKINESS_BY_IGNORING_SECURITY_DOMAINS, true);
-		extraCapabilities.setCapability("ignoreProtectedModeSettings", true);
-		super.beforeSuite();
-	}
+    @BeforeSuite(alwaysRun = true)
+    public void beforeSuite() throws InvalidBrowserException {
+        extraCapabilities = new DesiredCapabilities();
+        extraCapabilities.setCapability(InternetExplorerDriver.INTRODUCE_FLAKINESS_BY_IGNORING_SECURITY_DOMAINS, true);
+        extraCapabilities.setCapability("ignoreProtectedModeSettings", true);
+        super.beforeSuite();
+    }
 ```

```

Comment:
that beforeClass code you removed should be put in around here with an explanation, similar to the above